WBooks/Screens/BaseViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  BaseViewController.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 22/01/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class BaseViewController: UIViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.navigationBar.barStyle = .blackTranslucent
+        setNavigationBar()
+        UINavigationBar.appearance().titleTextAttributes = [NSAttributedStringKey.foregroundColor:UIColor.white]
+    }
+    
+    func setTitle(headerTitle: String? = nil) {
```

Comment:
Is the `? = nil` necessary?

Reply:
Im thinking that i can use for set no title by default but if i never call set title i'll never get a title so its unnecessary right now 😆